tests/test_main.py

```diff
@@ -5,9 +5,9 @@
 from io import StringIO
 
 from db.models import Actor, Genre, Movie, MovieSession, CinemaHall
-from services.movie import get_movies, get_movie_by_id, create_movie
-from services.cinema_hall import get_cinema_halls, create_cinema_hall
-from services.movie_session import (
+from db.services.movie import get_movies, get_movie_by_id, create_movie
```

Comment:
Do you really need to adjust the tests?

Reply:
No, it's not necessary, but I can't run tests without it. I tried to solve this, but I didn't succeed

Reply:
will the tests on github pass if you remove it?

Reply:
No, I checked

Reply:
Environment on github is the same for everyone, and tests pass without any adjustments

Reply:
can you push code without adjustments to tests and lets see what error exactly you will get

Reply:
Done

Reply:
services dir is inside of db

Reply:
Ah, I did not correctly understand the condition for creating the package